src/Logging/JUnit/Reader.php

```diff
@@ -107,12 +107,12 @@ public function getFeedback(): string
             . str_repeat('S', $this->suite->skipped)
             . str_repeat(
                 '.',
-                $this->suite->tests
-                - $this->suite->errors
-                - $this->suite->warnings
-                - $this->suite->failures
-                - $this->suite->risky
-                - $this->suite->skipped,
+                ($successTimes = $this->suite->tests
+                    - $this->suite->errors
+                    - $this->suite->warnings
+                    - $this->suite->failures
+                    - $this->suite->risky
+                    - $this->suite->skipped) >= 0 ? $successTimes : 0,
```

Comment:
Would be better to use wrap this in `max` instead?

```suggestion
                max(0, $this->suite->tests
                    - $this->suite->errors
                    - $this->suite->warnings
                    - $this->suite->failures
                    - $this->suite->risky
                    - $this->suite->skipped),
```